src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -195,6 +195,10 @@
     "NAME": "Fresh Adventure",
     "DESCRIPTION": "Deal !D! damage. NL Remember stsjorbsmod:Diligence."
   },
+  "stsjorbsmod:Frostbite": {
+    "NAME": "Frostbite",
+    "DESCRIPTION": "Deal !D! damage. NL If Fatal, stsjorbsmod:Consume this."
```

Comment:
The sheet's description here is inclusive of minion kills; I like the idea you did in Siphon where you made "works on minions" just an argument to the action, so we can change this easily later.